src/open_inwoner/cms/cases/views/status.py

```diff
@@ -541,8 +544,11 @@ def get_statuses_data(
         return [
             {
                 "date": s.datum_status_gezet,
-                "label": lookup.from_glom(
-                    s, "statustype.omschrijving", default=_("No data available")
+                "label": (
+                    s.statustype.statustekst
+                    or lookup.from_glom(
+                        s, "statustype.omschrijving", default=_("No data available")
+                    )
```

Comment:
It feels inconsistent to use glom for one attribute but then use attribute access for the other.

For compactness you can also modify the `from_glom()` helper (or add a specific method) to also accept a tuple/list of multiple gloms (and one default)